Tests/MongoSwiftSyncTests/SpecTestRunner/CodableExtensions.swift

```diff
@@ -64,33 +64,54 @@ extension TransactionOptions: Decodable {
     }
 }
 
-extension ReadPreference.Mode: Decodable {}
+extension ReadPreference.Mode: Decodable {
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.singleValueContainer()
+        var string = try container.decode(String.self)
+        // spec tests capitalize first letter of mode, so need to account for that.
```

Comment:
the spec tests changed how they specify the mode too, so a custom decodable initializer was also necessary here.